resources/fetching/fetchers/azure/batch_fetcher.go

```diff
@@ -57,29 +57,58 @@ func (f *AzureBatchAssetFetcher) Fetch(ctx context.Context, cMetadata fetching.C
 		}
 
 		if len(assets) == 0 {
+			if assetType == inventory.BastionAssetType {
```

Comment:
I think that we need the same behavior for Activity Logs as well.
Because if we do not have any Activity Logs configured we would like to fail rules that should verify that Activity Logs are configured.